Mathlib/Order/Monotone/Basic.lean

```diff
@@ -626,6 +626,13 @@ theorem StrictAnti.isMin_of_apply (hf : StrictAnti f) (ha : IsMax (f a)) : IsMin
     (hf hb).not_isMax ha
 #align strict_anti.is_min_of_apply StrictAnti.isMin_of_apply
 
+lemma StrictMono.add_le {f : ℕ → ℕ} (hf : StrictMono f) (m n : ℕ) : m + f n ≤ f (m + n)  := by
```

Comment:
This is true for some other types as well (e.g., for `Int`, if you assume `0 ≤ m`), so I suggest that you add "_nat" to the name of this lemma.

Reply:
Done.